pom.xml

```diff
@@ -178,7 +164,6 @@
         <plugin>
             <groupId>org.jacoco</groupId>
             <artifactId>jacoco-maven-plugin</artifactId>
-            <version>0.8.10</version>
```

Comment:
Is the version pinned in a parent or what is the reason behind this change?

Reply:
Yes: https://github.com/INRIA/spoon/pull/4748/files#diff-da6c5ffbf044fe3aff223deb941b61a7cb0506bbc2e2e82e4b15589ec02bbf83R316-R320